test/runtests.jl

```diff
@@ -24,6 +24,10 @@ import Impute:
     mask = map(!ismissing, a)
     ctx = Context(; limit=0.2)
 
+    @testset "Equality" begin
+        @test Fill(; value=mean, context=ctx) == Fill(; value=mean, context=ctx)
```

Comment:
is it worth something like?
```julia
testset "Equality $T" for T in InteractiveUtils.subtypes(Imputor)
        @test T() == T()
end
```

Reply:
I suppose, but I'd rather not use InteractiveUtils.